docs/development_suite/clusters-management/vendors-runtime-services.md

```diff
@@ -1,11 +1,16 @@
 ---
 id: vendors-runtime-services
-title: Vendors & Runtime services
-sidebar_label: Vendors & Runtime services
+title: Clusters Overview
+sidebar_label: Clusters
 ---
 
-When the user [connects a new cluster](/getting-started/handbooks/company/connect-and-manage-cluster.md#connect-a-cluster), they are required to enter information about the cluster's vendor and runtime service.
+Mia-Platform Console allows authorized users to manage the Kubernetes clusters on which projects are deployed. 
+
+## Supported clusters
```

Comment:
Non sarebbe meglio `Supported vendors and runtime services for a cluster` eventualmente?

Reply:
Changed